src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,14 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
```suggestion
export const CommentInfo = ({ comment: { name, email, body } }) => (
```
In this way, you can do destructuring assignment in nested properties

![image](https://github.com/user-attachments/assets/44fe0ec2-2479-4cb9-9792-a8791cf14bd7)